administration/src/components/generation/GenerationForm.tsx

```diff
@@ -72,10 +72,10 @@ const GenerationForm = (props: Props) => {
 
     const allCardsValid = cardCreationModels.reduce((acc, model) => acc && isValid(model), true)
 
+    usePrompt("Falls Sie fortfahren, werden alle Eingaben verworfen.", cardCreationModels.length !== 0);
+    
     return (
         <>
-            <Prompt message={"Falls Sie fortfahren, werden alle Eingaben verworfen."}
-                    when={cardCreationModels.length !== 0}/>
             <ButtonBar stickyTop={0}>
```

Comment:
Why are react routing libraries so shitty and unstable...

Reply:
My words...

Loving Flutter/Angular for that.